starknet_devnet/starknet_wrapper.py

```diff
@@ -269,7 +269,15 @@ def get_transaction_trace(self, transaction_hash:str):
 
         tx_hash_int = int(transaction_hash, 16)
         if tx_hash_int in self.__transaction_wrappers:
-            return self.__transaction_wrappers[tx_hash_int].trace
+            status = self.__transaction_wrappers[tx_hash_int].transaction["status"]
+            trace = self.__transaction_wrappers[tx_hash_int].trace
+
+            if not trace:
```

Comment:
When could this be true? Are you counting on `trace` to potentially be `None`, or it not even being a property of the origin object at all?